shared/src/main/scala/coop/rchain/shared/PathOps.scala

```diff
@@ -38,4 +42,49 @@ object PathOps {
         FileVisitResult.CONTINUE
       }
     }
+
+  implicit class PathDelete(path: Path) {
+    def delete[F[_]: Sync: Log](): F[Unit] = {
+      import java.io.File
+      import java.util.Comparator
+
+      import scala.collection.JavaConverters._
+
+      import cats.implicits._
+
+      def deleteFile(file: File)(implicit logSource: LogSource): F[Unit] =
+        for {
+          deleted <- Sync[F].delay(file.delete)
```

Comment:
what if file.delete throws?

Reply:
this is an internal function used only in `delete0()`. And `delete0()` is called once `delete0().attempt`